docs/content/preview/explore/indexes-constraints/partial-index-ysql.md

```diff
@@ -54,35 +57,35 @@ SELECT * FROM customers LIMIT 3;
 ```output
 customer_id |       company_name        |  contact_name  |    contact_title    |           address           |   city    | region | postal_code | country |     phone      |      fax
```

Comment:
SQL output tables are weird. Need leading spaces on this and the ones you removed
```suggestion
 customer_id |       company_name        |  contact_name  |    contact_title    |           address           |   city    | region | postal_code | country |     phone      |      fax
```